relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -476,6 +489,20 @@ mod tests {
         "ListAppViewController"
     );
 
+    span_description_test!(
+        span_description_file_write_keep_extension_only,
+        "data.data (42 KB)",
+        "file.write",
+        "data"
+    );
+
+    span_description_test!(
+        span_description_file_read_keep_extension_only,
+        "Info.plist",
+        "file.read",
+        "plist"
```

Comment:
@shruthilayaj is this the desired format, or do we need something like `*.plist`?

Reply:
yes please! can we do `*.plist`, `*.data` etc

Reply:
`*` in case there's no extension?

Reply:
I think we don't extract description for these - so null description, some of the examples i've seen without extensions have had pretty bad descriptions

Reply:
sorry or `*` whatever works actually, i didn't read your question properly